website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,105 @@
+
+
+#### Issues: 
+Issues are used to track bugs, feature requests, and other tasks in a project. When you find a bug or want to propose a new feature, you can create an issue in the project's issue tracker. This allows you to collaborate with other contributors and track the progress of your task. Many open source projects have "Good first issue" or "Help wanted" labels attached to certain issues, which are suitable for new contributors or those looking to make their first contribution. 
+
+These issues are typically well-defined and relatively easy to implement, and can provide a great opportunity for contributors to get started with the project and learn the workflow. Look for these labels in the issue tracker and consider starting with them if you are new to the project or open source contribution in general.
+
+#### Pull Requests: 
+Pull requests are a way to propose changes to a project. When you submit a pull request, you're asking the project maintainers to review your code or document changes and consider merging it into the main codebase. To create a pull request, you'll need to fork the project, make your changes, and then submit the pull request through the project's website or Git hosting platform.
+
+There are many online resources available to help you learn about Git, pull requests, and issues. Look for online tutorials, documentation, and forums where you can ask questions and get help from other contributors.
+
+### CNCF Resources
+**[The CNCF TAG (Technical Advisory Group)](https://github.com/cncf/tag-contributor-strategy/)**  is a group of experienced technical contributors who are dedicated to providing guidance and support to other contributors within the CNCF ecosystem. The main aim of the TAG is to ensure that all projects within the CNCF adhere to the highest technical standards, and that they are maintained and developed in a consistent and sustainable way. This includes advising on best practices for project governance, technical architecture, security, and interoperability, among other areas.
+The TAG also provides a platform for cross-project collaboration and knowledge-sharing, helping to foster a sense of community and cohesion within the CNCF ecosystem. They work closely with project maintainers, contributors, and other community members to provide guidance and support, and to ensure that all projects within the CNCF are able to thrive and evolve in a collaborative and inclusive manner.
```

Comment:
"The TAG also provides a platform" --> the platform here makes me feel like there is a website or a tool or something like that. I can't come up with a better word though 🤷

Reply:
what about forum?
"The TAG provides a forum for cross-project collaboration....."